### DISCUSSION TITLE
Me banearon sin hacer nada

### DISCUSSION THREAD

**Discussion by kukoPy** [chromiecraft/chromiecraft#6123] [General]
cambie el idioma del juego de EU a ES y al ingresar me dice que estoy baneado pero que puede ser desbaneada ...!

**Comment by 55Honey** [comment]
Hello @kukoPy ,
Thanks for reaching out.
You're not connecting to the ChromieCraft server.

Please go to the `data/esES` (or whatever language you're using) directory and change the file `realmlist.wtf` to `logon.chromiecraft.com`.

**Reply by kukoPy** [reply]
Genial..! muchas gracias!! que despistado por cierto..!
pero para evitar que otro latino caiga en lo mismo podemos dejar un parche en ES ya incluido en la pag..! a modo sugerencias!
desde ya muchas gracias!!

**Reply by 55Honey** [reply]
You're welcome :-)
